code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -118,10 +118,10 @@ Doesn't work on other aliens/AI.*/
 	set desc = "Secrete tough malleable resin."
 	set category = "Alien"
 
-	if(powerc(55))
+	if(powerc(55, 1))
```

Comment:
why is this scoped on the carbon level
```dm
	else if(Y && (!isturf(src.loc) || istype(src.loc, /turf/space)))
		to_chat(src, "<span class='noticealien'>You can't place that here!</span>")
		return 0
```
xenos are really just something else

Reply:
It's because of xeno organs.  

Also use TRUE instead of 1 here

Reply:
why is that parameter named Y

Reply:
Only God knows